subprojects/docs/src/docs/userguide/dependencyManagementCustomizingDependencyResolutionBehavior.adoc

```diff
@@ -0,0 +1,308 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[customizing_dependency_resolution_behavior]]
+== Customizing the dependency resolution behavior
```

Comment:
Same comment about filename and title case

Reply:
done